src/System.IdentityModel.Tokens.Jwt/JwtHeader.cs

```diff
@@ -57,13 +57,20 @@ public JwtHeader(SigningCredentials signingCredentials)
         {
             if (signingCredentials == null)
             {
-                this[JwtHeaderParameterNames.Alg] = SecurityAlgorithms.NONE;
+                this[JwtHeaderParameterNames.Alg] = JwtAlgorithms.NONE;
             }
             else
             {
-                this[JwtHeaderParameterNames.Alg] = signingCredentials.Algorithm;
+                string algorithm;
+                if (OutboundAlgorithmMap.TryGetValue(signingCredentials.Algorithm, out algorithm))
+                {
+                    this[JwtHeaderParameterNames.Alg] = algorithm;
+                }
+
                 if (!string.IsNullOrEmpty(signingCredentials.Key.KeyId))
+                {
```

Comment:
we are moving towards a coding style to not use braces { } for a single line statement. https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md